jbmc/regression/jbmc/exception-cleanup/test.desc

```diff
@@ -0,0 +1,14 @@
+CORE
+Test.class
+--function Test.main
+^EXIT=0$
+^SIGNAL=0$
+^VERIFICATION SUCCESSFUL$
+1 remaining after simplification$
+--
+^warning: ignoring
+--
+The unreachable function should be successfully marked unreachable by symex,
+but the final uncaught exception is not yet, as it requires symex to note that
+within a catch block @inflight_exception is definitely *not* null, which it
+can't just yet.
```

Comment:
Not sure I understand what that comment is here for. The test is already marked as `CORE`, so what difference should we see when the stated limitation is resolved?

Reply:
Then we'll resolve *all* conditions in symex. I'll amend to make that clearer.